### PR TITLE
Add docs on how to use w/ `rehype-sanitize`

### LINKED ISSUE #67: rehype-sanitize breaks math rendering
### Initial checklist

- [X] I read the [support docs](https://github.com/remarkjs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/remarkjs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/remarkjs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Aremarkjs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

remark-math@5.1.0, rehype-katex@6.0.1, rehype-sanitize@5.0.0

### Link to runnable example

https://codesandbox.io/s/react-markdown-debug-forked-3rbkc?file=/src/App.js

### Steps to reproduce

The above sandbox link should work!

<details>
 <summary>Code:</summary>
```jsx
import React from "react";
import Markdown from "react-markdown";
import rehypeKatex from "rehype-katex";
import rehypeSanitize from "rehype-sanitize";
import remarkMath from "remark-math";

import "katex/dist/katex.min.css";

const mathMarkdown = `
Lift($L$) can be determined by Lift Coefficient ($C_L$) like the following
equation.

$$
L = \\frac{1}{2} \\rho v^2 S C_L
$$
`;

export default function App() {
  return (
    <>
      <h1>Without sanitize</h1>
      <Markdown rehypePlugins={[rehypeKatex]} remarkPlugins={[remarkMath]}>
        {mathMarkdown}
      </Markdown>
      <h1>With sanitize</h1>
      <Markdown
        rehypePlugins={[rehypeKatex, rehypeSanitize]}
        remarkPlugins={[remarkMath]}
      >
        {mathMarkdown}
      </Markdown>
    </>
  );
}
```
</details>

### Expected behavior

rehype-katex should still be able to correctly render math after rehype-sanitize runs. According to https://github.com/remarkjs/remark-math#security its advised to add rehype-sanitize if accepting user input. Even if safe without sanitize, I still need rehype-sanitize since I'm also using rehype-raw and rehype-autolink-headings.

Note, this breaks regardless of the order of plugins are added.

### Actual behavior

Math does not correctly render.

### Runtime

Other (please specify in steps to reproduce)

### Package manager

yarn v1

### OS

macOS

### Build and bundle tools

Webpack

### PR DESCRIPTION
**Pull request by maclockard**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

Mention in the docs how to safely configure `rehype-sanitize` to allow the required class names for remark-math

Fixes #67

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -99,6 +99,38 @@ so any vulnerability in KaTeX can open you to a
 
 Always be wary of user input and use [`rehype-sanitize`][rehype-sanitize].
 
+If you are using [`rehype-sanitize`][rehype-sanitize], but fully trust that
+[KaTeX][] doesn't have any XSS vulnerabilities, you can allow list the required
```
Comment (wooorm) [inline readme.md:103]:
```suggestion
[KaTeX][] doesn’t have any XSS vulnerabilities, you can allow list the required
```

File: readme.md [hunk 2]
```diff
@@ -99,6 +99,38 @@ so any vulnerability in KaTeX can open you to a
...
+      ["className", "math", "math-display"],
+    ],
+    span: [
+      ["className", "math", "math-inline"],
+    ],
+  },
+};
+```
+
+And applying the `rehype-katex` plugin _after_ the [`rehype-sanitize`][rehype-sanitize] one like so:
```
Comment (wooorm) [inline readme.md:122]:
```suggestion
And applying the `rehype-katex` plugin *after* the [`rehype-sanitize`][rehype-sanitize] one like so:
```

running `npm install` and `npm test` might catch some prose stuff

**Review by wooorm — COMMENTED** [review]
nice! Just some nits.

**Review by wooorm — COMMENTED** [review]
some style nits, otherwise good to go!

**Review by ChristianMurphy — APPROVED** [review]

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks Mac!

**Pull request merged**